tests/test_moduletool.py

```diff
@@ -667,6 +667,34 @@ def test_project_freeze(modules_dir, modules_repo, capsys):
     )
 
 
+def test_project_freeze_disk(modules_dir, modules_repo, capsys):
+    coroot = install_project(modules_dir, "modA")
+
+    app(["project", "freeze"])
+
+    out, err = capsys.readouterr()
+
+    assert os.path.getsize(os.path.join(coroot, "project.yml")) != 0
+    assert len(err) == 0, err
+
+    with open(os.path.join(coroot, "project.yml"), "r") as fh:
+        assert (fh.read()
+        == """name: modA
+license: Apache 2.0
+version: 0.0.1
+modulepath: libs
+downloadpath: libs
+repo: %s
+requires:
+- modB ~= 3.2
+- modC ~= 3.2
```

Comment:
According to the `modules_repo` fixture, modC should have version 3.0 instead of 3.2

Reply:
```
c = make_module_simple_deps(reporoot, "C", ["E", "F", "E::a"], version="3.0")
add_file(c, "model/a.cf", "import modI", "add mod C::a", "3.2")
```

so the highest is 3.2

Reply:
Ok. I didn't notice the second line.